docs/source/conf.py

```diff
@@ -3497,33 +3577,20 @@ def linkcode_resolve(domain, info):
 #
 #
 
-html_theme = "pytorch_sphinx_theme"
-html_theme_path = [pytorch_sphinx_theme.get_html_theme_path()]
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
 
-html_theme_options = {
-    "pytorch_project": "docs",
-    "canonical_url": "https://pytorch.org/docs/stable/",
-    "collapse_navigation": False,
-    "display_version": True,
-    "logo_only": True,
-    "analytics_id": "GTM-T8XT4PS",
-}
-
-html_logo = "_static/img/pytorch-logo-dark-unstable.png"
-if RELEASE:
-    html_logo = "_static/img/pytorch-logo-dark.svg"
-
-
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ["_static"]
 
-html_css_files = ["css/jit.css", "css/custom.css"]
+html_css_files = [
+    "css/jit.css",
+    "css/custom.css",
+    "https://cdn.jsdelivr.net/npm/katex@0.10.0-beta/dist/katex.min.css",
```

Comment:
This is a new dependency?

Reply:
it's not new. It was here: https://github.com/pytorch/pytorch/blob/main/docs/source/conf.py#L3651-L3657 - this was the old way to load css. Now it's done with html_css_files